src/sentry/static/sentry/app/views/performance/transactionSummary/tagExplorer.tsx

```diff
@@ -0,0 +1,337 @@
+import React from 'react';
+import {browserHistory} from 'react-router';
+import styled from '@emotion/styled';
+import {Location} from 'history';
+
+import DropdownButton from 'app/components/dropdownButton';
+import DropdownControl, {DropdownItem} from 'app/components/dropdownControl';
+import Duration from 'app/components/duration';
+import GridEditable from 'app/components/gridEditable';
+import Link from 'app/components/links/link';
+import Pagination from 'app/components/pagination';
+import {t} from 'app/locale';
+import space from 'app/styles/space';
+import {Organization, Project} from 'app/types';
+import EventView from 'app/utils/discover/eventView';
+import {formatAbbreviatedNumber} from 'app/utils/formatters';
+import SegmentExplorerQuery, {
+  TableDataRow,
+  TopValue,
+} from 'app/utils/performance/segmentExplorer/segmentExplorerQuery';
+import {decodeScalar} from 'app/utils/queryString';
+import {stringifyQueryObject, tokenizeSearch} from 'app/utils/tokenizeSearch';
+import CellAction from 'app/views/eventsV2/table/cellAction';
+import {TableColumn} from 'app/views/eventsV2/table/types';
+
+const COLUMN_ORDER = [
+  {
+    key: 'key',
+    name: 'Key',
+    width: -1,
+    column: {
+      kind: 'field',
+    },
+  },
+  {
+    key: 'topValues',
+    name: 'Values',
+    width: -1,
+    column: {
+      kind: 'field',
+    },
+  },
+];
+
+const HEADER_OPTIONS: DropdownOption[] = [
+  {
+    label: 'Slowest Tag Values',
+    value: '-aggregate',
+  },
+  {
+    label: 'Fastest Tag Values',
+    value: 'aggregate',
+  },
+  {
+    label: 'Most Frequent Tag Values',
+    value: '-count',
+  },
+];
+
+const DURATION_OPTIONS: DropdownOption[] = [
+  {
+    label: 'transaction.duration',
+    value: 'duration',
+  },
+  {
+    label: 'measurements.lcp',
+    value: 'measurements[lcp]',
+  },
+];
+
+const handleTagValueClick = (location: Location, tagKey: string, tagValue: string) => {
+  const queryString = decodeScalar(location.query.query);
+  const conditions = tokenizeSearch(queryString || '');
+
+  conditions.addTagValues(tagKey, [tagValue]);
+
+  const query = stringifyQueryObject(conditions);
+  browserHistory.push({
+    pathname: location.pathname,
+    query: {
+      ...location.query,
+      query: String(query).trim(),
+    },
+  });
+};
+
+const renderBodyCell = (
+  parentProps: Props,
+  column: TableColumn<keyof TableDataRow>,
+  dataRow: TableDataRow
+): React.ReactNode => {
+  const value = dataRow[column.key];
+  const {location} = parentProps;
+
+  if (Array.isArray(value)) {
+    return (
+      // TODO(Kevan): Remove ts any
+      <CellAction column={column} dataRow={dataRow as any} handleCellAction={() => {}}>
```

Comment:
Why is there a cell action that does nothing?

Reply:
Ah. I was going to put cell actions in for the times but dropped it. Good catch.